server/api/documents.js

```diff
@@ -15,14 +15,17 @@ const authDocumentForUser = (ctx, document) => {
 const router = new Router();
 
 router.post('documents.list', auth(), pagination(), async ctx => {
-  let { sort = 'updatedAt', direction } = ctx.body;
+  let { sort = 'updatedAt', direction, collectionId } = ctx.body;
```

Comment:
so far I just left out `id`s from the fields in the API, so you pass the collection ID with `collection` field. Once this is changed, could you also append to public API docs?

Reply:
Oh yea, I'm really unused to that so I forgot.